CHANGES.rst

```diff
@@ -33,6 +33,9 @@ General
   https://libcloud.readthedocs.io/en/latest/other/using-http-proxy.html
   (GITHUB-1314)
   [Jim Liu - @hldh214, Tomaz Muraus]
+- Specify the minimum required version of requests.
```

Comment:
Please also add a note and reference the github issue on why we need to specify the minimum version (aka issues on RHEL).

Reply:
Note that setting the minimum requests version to `2.5.0` is not directly related to the RHEL issue in https://github.com/apache/libcloud/issues/1322 (the root cause of that issue is differences in the code of requests 2.6.0 shipped in RPM vs PyPI). However, the investigation of that issue led me to discover that `requests<=2.4.0` crashes on the Azure Storage driver which is what inspired this PR. I've made this more explicit in 5fe2332.